packages/css/src/components/progress-list/progress-list.scss

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@use "../../common/hyphenation" as *;
+@use "../../common/text-rendering" as *;
+
+@mixin reset-ol {
+  list-style-type: none;
+  margin-block: 0;
+  padding-inline-start: 0;
+}
+
+.ams-progress-list__connector {
+  background-color: var(--ams-progress-list-connector-background-color);
+  block-size: 100%;
+  grid-column: 1; // This is used to position the connector in the middle of the marker.
+  grid-row: 1; // This is used to position the connector in the middle of the marker.
+  inline-size: var(--ams-progress-list-connector-inline-size);
+
+  &-heading-2 {
```

Comment:
We don’t use the parent selector of Sass to concatenate classes – this makes classes less easy to find in your IDE.

Also, the ‘heading-x’ variants must be separated with a double hyphen in BEM: `.ams-progress-list__marker--heading-2`